README.md

```diff
@@ -117,36 +131,29 @@ You can see the details in
 
 ### Authorization
 
-To see authorization in action in this example, all requests to the endpoint
-`/users/{userId}/orders` (i.e, `UserOrderController`) are validated for specific
-user access.
-
-Endpoint authorization is done using `@loopback/authorization` package and the
-provisions in it are implemented using
-[Casbin](https://github.com/casbin/casbin) configurations.
+Endpoint authorization is done using
```

Comment:
Link in authorization intro not producing a visible blue link:

![image](https://user-images.githubusercontent.com/6864736/73546063-8c27c600-440a-11ea-8b38-8b9ef8f28a04.png)

Reply:
The single quote should be inside [ ]: 
[`@loopback/authorization`](https://github.com/strongloop/loopback-next/tree/master/packages/authorization)